dotnet/src/SemanticKernel.Abstractions/Contents/BinaryContent.cs

```diff
@@ -106,7 +106,7 @@ public BinaryContent(
     /// <param name="mimeType">The mime type of the content</param>
     public BinaryContent(
         ReadOnlyMemory<byte> data,
-        string? mimeType)
+        string mimeType)
```

Comment:
This is a breaking change, for now I would not change SK contents to require a `mimeType`.

```suggestion
        string mimeType)
```